docs/ui-coverage/configuration/views.mdx

```diff
@@ -72,29 +68,30 @@ An array of named URL parameters used to further subdivide the URLs into multipl
 #### Visited URLs
 
 ```
-https://cypress.io/users/alice
-https://cypress.io/users/alice?foo=bar
-https://cypress.io/users/bob
-https://cypress.io/users/bob#baz
+https://www.my-app.com/users/alice
+https://www.my-app.com/users/bob
+https://www.my-app.com/users/bob#settings
+https://www.my-app.com/users?assigned=true
 ```
 
 #### Views shown in UI
 
 ```
-https://cypress.io/users/*
+my-app.com/users
+my-app.com/users/*
```

Comment:
Either

```
https://www.my-app.com/users
https://www.my-app.com/users/*
```

or

```
www.my-app.com/users
www.my-app.com/users/*
```

or

```
/users
/users/*
```

stripping only the `www.` does not make sense / match our logic